software/chhoto-url.yml

```
@@ -0,0 +1,13 @@
+name: Chhoto URL
+website_url: https://github.com/SinTan1729/chhoto-url
+description: Simple, lightning-fast URL shortener with no bloat (fork of simply-shorten).
+licenses:
+  - identifier: MIT
+    name: MIT License
+    url: https://spdx.org/licenses/MIT.html
```

```suggestion
  - MIT
```
this just needs MIT, not the name, identifier or URL as it will be automatically applied to the list.

---

Yes, it was my misunderstanding. Thanks for the suggestion.